### PR TITLE
Add brief description of header files for doxygen

### PR DESCRIPTION
**Pull request by junmin-kim**

### CODE REVIEW

File: os/include/unistd.h [hunk 1]
```diff
@@ -51,6 +51,7 @@
  ****************************************************************************/
 /**
  * @defgroup UNISTD_KERNEL UNISTD
+ * @brief Provides APIs for Standard symbolic constants and Types
```
Comment (chanijjani) [inline os/include/unistd.h]: symbolic constants 요기 그냥 소문자로 시작하게 두어도 상관없는건가요~?

Comment (junmin-kim) [inline os/include/unistd.h]: 1. Standard Symbolic Constants and Types
2. Standard Symbolic constants and Types
3. Standard Symbolic Constants And Types
Which one should I use?

Comment (chanijjani) [inline os/include/unistd.h]: I'm not sure but how about applying the option '1' for consistent presentations? :)

**Comment by junmin-kim** [comment]
@chanijjani I fixed it. Please review it again.

**Pull request merged**